packages/flutter/test/material/dropdown_test.dart

```diff
@@ -63,13 +63,13 @@ Widget buildFrame({
             isDense: isDense,
             isExpanded: isExpanded,
             underline: underline,
-            items: items == null ? null : items.map<DropdownMenuItem<String>>((String item) {
+            items: items?.map<DropdownMenuItem<String>>((String item) {
               return DropdownMenuItem<String>(
                 key: ValueKey<String>(item),
                 value: item,
                 child: Text(item, key: ValueKey<String>(item + 'Text')),
               );
-            }).toList(),
+            })?.toList(),
```

Comment:
Same thing here

Reply:
this one i think the original is more readable. It's very confusing to have the random question mark multiple lines down this expression.